src/standard/index.d.ts

```diff
@@ -37,7 +37,17 @@ export declare interface StripeBackendAPI {
    * @returns a Promise that resolves on success and rejects on failure.
    *     Any error should be reported as a string that can be displayed to the user.
    */
-  completeCharge(sourceID: string, amount: number, shippingMethod?: StripeShippingMethod, shippingAddress?: StripeAddress): Promise<void>;
+  capturePayment(sourceID: string, amount: number, shippingMethod?: StripeShippingMethod, shippingAddress?: StripeAddress): Promise<void>;
+
+
```

Comment:
Should we keep `completeCharge` for backward compatibility?

(also, extra blank line)

Reply:
Yes good point

Reply:
I'd rather mention it in the Changelog, but I don't think it's a breaking change for anyone as it's wrapped into specific methods on every calls. 
Only case would be if you hit `stripeService. completeCharge` directly (and I don't think it's happening out of a sandbox mode)